src/main/webapp/WEB-INF/static/js/trialmanager/subobservations/subObservationSet.js

```diff
@@ -1458,7 +1458,7 @@
 							render: function (data, type, full, meta) {
 								return '<a class="gid-link" href="javascript: void(0)" ' +
 									'onclick="openGermplasmDetailsPopopWithGidAndDesig(\'' +
-									full.gid + '\',\'' + full.designation + '\')">' + EscapeHTML.escape(data.value) + '</a>';
+									full.gid + '\')">' + EscapeHTML.escape(data.value) + '</a>';
```

Comment:
Since this component is already in AngularJS, use ng-click similar to this:

$scope.openGermplasmDetailsModal = function(gid) {
				germplasmDetailsModalService.openGermplasmDetailsModal(gid);
			}

columnsDef.push({
							targets: columns.length - 1,
							orderable: false,
							render: function (data, type, full, meta) {
								return '';
							},
							createdCell: function (td, cellData, rowData, rowIndex, colIndex) {
								$(td).append($compile('<a class="gid-link" href="javascript: void(0)" ' +
									'ng-click="openGermplasmDetailsModal(\'' +
									rowData.gid + '\')">' + EscapeHTML.escape(cellData.value) + '</a>')($scope));
							}
						});